src/Symfony/Component/Form/Extension/Core/DataTransformer/IntegerToLocalizedStringTransformer.php

```diff
@@ -40,6 +42,12 @@ public function __construct($scale = 0, $grouping = false, $roundingMode = self:
      */
     public function reverseTransform($value)
     {
+        $decimalSeparator = $this->getNumberFormatter()->getSymbol(\NumberFormatter::DECIMAL_SEPARATOR_SYMBOL);
+
+        if (\is_string($value) && false !== strpos($value, $decimalSeparator)) {
+            throw new TransformationFailedException(sprintf('The float "%s" is not a valid integer.', $value));
```

Comment:
Is it "The float..." or "A float..." here?
I mean is there a case where some kind of float is valid? if yes, a test is missing, isn't it?

Reply:
I agree with @OskarStark and probably the `The float ...` should be changed by `The value ...`?